### ISSUE TITLE
script tag cause bad page rendering

### ISSUE THREAD

**Issue by mozbugbox** [tats/w3m#151]
w3m doesn't render html files with self closed `<script>` tag like:
```
<script src="asource.js" />
```
If the script tag closed with "</script>", then the rendering works properly:
```
<script src="asource.js"></script>
```

A sample file uploaded:
[test.txt](https://github.com/tats/w3m/files/5331066/test.txt)

**Comment by DannyJJK** [comment]
This is invalid HTML, the script tag needs a </script>. It may work in other browsers as they "fix" the HTML, but this is invalid syntax

**Comment by bptato** [comment]
> This is invalid HTML, the script tag needs a </script>. It may work in other browsers as they "fix" the HTML, but this is invalid syntax

Indeed, it doesn't work in Firefox either. It is however valid xhtml, which w3m parses with the same parser as html, so perhaps OP ran into an xhtml page.
I've looked into basic xhtml support before (so such a tag would be accepted in xhtml documents) but I couldn't figure out how to; perhaps I'll try again in the future.
